test/test_torch.py

```diff
@@ -6021,6 +6021,73 @@ def test_pow(self, device):
             torch.pow(m1, 1, out=out)
             self.assertEqual(out, m1)
 
+    @dtypes(*list(product(torch.testing.get_all_dtypes(include_bool=False),
```

Comment:
Add some entries to common_methods_invocations.py's method_tests(), too, like the pow entries:

https://github.com/pytorch/pytorch/blob/78b95b6204809822def6dd1b06d03cf002cd30c5/torch/testing/_internal/common_methods_invocations.py#L563

Reply:
I have add some entries like the `pow`, and them worked on `float_power`. 
But there are still some errors for autodiff, just like:
```
/root/anaconda3/envs/xue/lib/python3.7/site-packages/torch/testing/_internal/jit_utils.py:687: in wrapper
    fn(*args, **kwargs)
test/test_jit.py:15721: in check
    self.assertAutodiffNode(traced_fn.last_graph, should_autodiff_node, autodiff_nodes, fusible_nodes)
/root/anaconda3/envs/xue/lib/python3.7/site-packages/torch/testing/_internal/jit_utils.py:350: in assertAutodiffNode
    self.assertEqual(should_autodiff_node, found_all_nonfusible_nodes and found_all_fusible_nodes)
/root/anaconda3/envs/xue/lib/python3.7/site-packages/torch/testing/_internal/common_utils.py:1173: in assertEqual
    self.assertTrue(x == y, msg=msg)
E   AssertionError: False is not true
```
I am not familiar with autodiff, so I turn the autodiff_check to false temporarily.
I'm not sure does it achieve the goal...

Reply:
That's OK for now. In the near future this will be much clearer.